vmdb/lib/workers/schedule_worker.rb

```diff
@@ -233,6 +216,23 @@ def schedules_for_scheduler_role
       @queue.enq :storage_scan_timer
     end
 
+    schedules_for_ems_refresh
+  end
+
+  def schedules_for_ems_refresh
+    config = VMDB::Config.new("vmdb").config.fetch(:ems_refresh, {})
+
+    ExtManagementSystem.leaf_subclasses.each do |klass|
+      every   = config.fetch_path(klass.ems_type.to_sym, :refresh_interval)
+      every ||= config.fetch(:refresh_interval, 24.hours)
```

Comment:
It's there... https://github.com/Fryguy/manageiq/blob/d606929b49d0b49731a2877b51dfbb653cb3dad9/vmdb/config/vmdb.tmpl.yml#L79